mlflow/models/cli.py

```diff
@@ -35,15 +35,15 @@ def commands():
 @cli_args.INSTALL_MLFLOW
 @cli_args.ENABLE_MLSERVER
 def serve(
-    model_uri,
-    port,
-    host,
-    timeout,
-    workers,
-    no_conda,  # pylint: disable=unused-argument
-    env_manager=None,
-    install_mlflow=False,
-    enable_mlserver=False,
+        model_uri,
```

Comment:
@anuarkaliyev23 Thanks for the PR! Can you revert unrelated changes?

Reply:
Yes, I'm sorry, my IDE seems to change indents and I didn't notice.
Fixed